src/scripts/seqbox_utils.py

```diff
@@ -278,6 +303,12 @@ def get_pangolin_result(pangolin_result_info):
         sys.exit(1)
 
 
+def check_nans(value):
```

Comment:
Seems like this is going return None when value is not NaN (i.e. when isnan returns False).

So dates are numbers when we use pd.read_csv etc are they?

Can you add some comments explaining what is happening here?